screen/model/coreConnection.js

```diff
@@ -0,0 +1,143 @@
+
+module.exports = class CoreConnection {
+
+    constructor(address, onMessageCallback = null, reconnect = true, reconnectWait = 3000){
```

Comment:
I plan to make the CoreConnection in the client extend EventTarget instead of having a custom system of registering callbacks. I don't know if it makes sense for you to do it here if you only have one callback, but I wanted to share just in case.

Reply:
Created #38 for this.